action/template_test.go

```diff
@@ -19,30 +21,44 @@ func TestTemplate(t *testing.T) {
 	defer func() { log.Stdout = o }()
 
 	// TOML
-	Template("", tpl, val)
+	Template("", tpl, val, false)
 	if out.String() != "Hello World!\n" {
 		t.Errorf("Expected Hello World!, got %q", out.String())
 	}
 
+	// force false
+	err = Template(tpl, val, "", false)
+	if err == nil {
```

Comment:
FYI for the future, you can combine this and the previous line with `if err = Template(...); err == nil`